cache-config/t3c/t3c.go

```diff
@@ -48,18 +48,18 @@ func main() {
 	flagHelp := getopt.BoolLong("help", 'h', "Print usage information and exit")
 	getopt.Parse()
 	if *flagHelp {
-		fmt.Println(usageStr())
+		log.Errorln(usageStr())
```

Comment:
Loggers were never initialized, needs a call to `log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)`

Reply:
Added log.Init()